cmd/promremotecli.go

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
nit: Perhaps just rename this to `main.go`?  It seems cleaner to run it that way, i.e. `go run cmd/main.go -t foo:bar -d now`